### DISCUSSION TITLE
Entra&Esci living now e zigbee

### DISCUSSION THREAD

**Discussion by developerGM** [indomus/forum#2749] [HUB: Home Assistant]
Ciao a tutti, 
ho integrato su HA il sistema della Bticino living now tramite homekit per non perdere l'utilizzo dell'app originale della Bticino. 
I server della Bticino parecchie volte sono down e la comunicazione con HA spesso non funziona. 
Vorrei quindi svincolarmi completamente dai sistemi della Bticino collegando tutti gli interruttori connessi tramite protocollo zigbee.
Il problema è il pulsante Entra&esci... C'è un modo per collegarlo ed utilizzarlo su HA?

**Comment by essegi65** [comment]
> Ciao a tutti, ho integrato su HA il sistema della Bticino living now tramite homekit per non perdere l'utilizzo dell'app originale della Bticino. I server della Bticino parecchie volte sono down e la comunicazione con HA spesso non funziona. Vorrei quindi svincolarmi completamente dai sistemi della Bticino collegando tutti gli interruttori connessi tramite protocollo zigbee. Il problema è il pulsante Entra&esci... C'è un modo per collegarlo ed utilizzarlo su HA?

Da quello che so il pulsante entra/esci lo puoi vedere se colleghi il sistema living now ad HA tramite Homekitcontroller, ma non espone entità che puoi comandare. Se colleghi i dispositivi a HA con Z2M, come ho fatto anche io insieme a tanti altri, il comando entra/esci lo puoi simulare creando automazioni addirittura collegandoci la presenza o meno delle persone tramite GPS o wi-fi

**Reply by developerGM** [reply]
Esatto, infatti vedo l'entità ma non mi fa fare operazione...neanche in lettura quando clicco su Entra o Esci per poterci attaccare magari qualche altra automazione su HA.
Ho già collegato le prese connessa tramite Zigbee2MQTT e vanno bene...volevo evitare di scollegare anche tutte le luci dal sistema Bticino per collegarle a Zigbee2MQTT solo per evitare di dover scollare dal muro l'Entra&Esci.
Ma a questo punto mi sa che farò cosi perchè molto spesso i server della Bticino vanno in down ed ho problemi a gestire le luci tramite Homekit su HA,

Collego tutto su Zigbee2MQTT, rimuovo l'entraesci e ci attacco uno switch zigbee che fa la stessa cosa dell'entraesci

**Reply by jovannig** [reply]
Io ho un problema analogo. Ho la living now col gateway bticino ed ho portato tutte le prese e accessori su HA tramite pero' ZHA.
Funziona tutto perfettamente, l'unico che non sono riuscito a portare su HA e' il comando entra esci. Ho provato a resettarlo in tutti i modi ma ZHA non riesce mai a vederlo. Adesso ho anche perso la connessione con l'app bticino perche' non riesco piu' ad accoppiarlo al gateway bticino. Potete aiutarmi? Grazie mille
